packages/driver/src/cy/chai.js

```diff
@@ -365,8 +452,8 @@ chai.use((chai, u) => {
     const fns = _.functions(chai.assert)
 
     _.each(fns, (name) => {
-      return fn[name] = function (...args) {
-        return chai.assert[name].apply(this, args)
+      return fn[name] = function () {
+        return chai.assert[name].apply(this, arguments)
```

Comment:
using `arguments` instead of the rest operator is more performant and, in my opinion, more readable